api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageProcessor.java

```diff
@@ -44,8 +48,8 @@ public void processAllMessages() {
     }
 
     private void processMessagesOfType(MessageType messageType) {
-        List<Message> messages = messageRepository.findAllMessagesByTypeOrderedByDate(messageType);
-        if (CollectionUtils.isEmpty(messages)) {
+        List<Message> messages = messageRepository.findAllMessagesByTypeOrderedByDate(messageType, PageRequest.of(0, messageProcessingLimit));
+        if (messages.isEmpty()) {
```

Comment:
Do we no longer have to deal with a null response?

Reply:
Yes, we do, I've changed it back.